.ci/pytorch/common_utils.sh

```diff
@@ -246,6 +246,7 @@ function install_torchrec_and_fbgemm() {
       git clone --recursive https://github.com/pytorch/fbgemm
       pushd fbgemm/fbgemm_gpu
       git checkout "${fbgemm_commit}"
```

Comment:
Git checkout has a --recurse-submodules arg, just use that.

Reply:
You learn something new every day... updated, thanks!